src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const headRow = document.querySelector('thead').firstElementChild;
+const tbody = document.querySelector('tbody');
+const data = [];
+
+for (const row of tbody.children) {
+  data.push([row.children[0].textContent,
+    row.children[1].textContent, +row.children[2].textContent,
+    +row.children[3].textContent.replace(/\$|,/g, '')]);
```

Comment:
```suggestion
  data.push({
  	salary: ... ,
  	name: ... ,
  });
```

better use objects, so it will be more clear what we are sorting